draft-ietf-tls-esni.md

```diff
@@ -896,8 +906,8 @@ first ClientHelloOuter as follows:
 If the client-facing server accepted ECH, it checks the second ClientHelloOuter
 also contains the "encrypted_client_hello" extension. If not, it MUST abort the
 handshake with a "missing_extension" alert. Otherwise, it checks that
-ClientECH.cipher_suite is unchanged, and that ClientECH.config_id and
-ClientECH.enc are empty. If not, it MUST abort the handshake with an
+ClientECH.cipher_suite and ClientECH.config_id are unchanged, and that
+ClientECH.enc is empty. If not, it MUST abort the handshake with an
```

Comment:
I believe this change needs to be reverted as well. From {#client-hrr}:

> The client then modifies the "encrypted_client_hello" extension in ClientHelloOuter as follows:
> 
>   - **config_id is replaced with the empty string.**
>   - cipher_suite is unchanged and contains the client's chosen HPKE cipher suite.
>   - enc is replaced with the empty string.
>   - payload is replaced with the value computed above.

Reply:
Fixed.